components/eam/chem_proc/inputs/pp_chemUCI_linozv3_mam4_resus_mom_soag_tag.in

```diff
@@ -192,7 +192,9 @@ CHEMISTRY
 [lhno4]     HO2NO2 + OH -> NO2 + H2O + O2                  ; 1.300E-12, 380
 [lhno3]     HNO3 + OH -> NO3 + H2O                         ; 2.400E-14, 460
 *   fix needed to add this bimolec rate, missing from the one below
-[uci6] HNO3 + OH -> NO3 + H2O
+[uci6] HNO3 + OH -> NO3 + H2O                              ; 2.400E-14, 460
+* uci6 coefficients assigned here are to avoid debug issue
+* the real coefficients will calculated in the model
```

Comment:
I'd suggest revising the comment to "uci6 reaction coefficients are place holders to avoid the floating invalid error with debug built. The actual uci6 rate is calculated by customized code with the equation below."

Reply:
@tangq changed